secant/Features/Home/HomeStore.swift

```diff
@@ -173,28 +175,39 @@ extension HomeReducer {
             return .none
 
         case .profile(.settings(.quickRescan)):
-            do {
-                try environment.SDKSynchronizer.rewind(.quick)
-            } catch {
-                // TODO [#221]: error we need to handle (https://github.com/zcash/secant-ios-wallet/issues/221)
-            }
             state.route = nil
-            return .none
+            return Effect.task {
+                do {
+                    try await environment.SDKSynchronizer.rewind(.quick)
+                    return .rewindDone
+                } catch {
+                    return .rewindFailed(.quickRescan)
+                }
+            }
 
         case .profile(.settings(.fullRescan)):
-            do {
-                try environment.SDKSynchronizer.rewind(.birthday)
-            } catch {
-                // TODO [#221]: error we need to handle (https://github.com/zcash/secant-ios-wallet/issues/221)
-            }
             state.route = nil
-            return .none
+            return Effect.task {
+                do {
+                    try await environment.SDKSynchronizer.rewind(.birthday)
+                    return .rewindDone
+                } catch {
+                    return .rewindFailed(.fullRescan)
+                }
+            }
```

Comment:
Same as above, fireAndForget is probably better than returning some Action only because task expect it.